src/cpu/probes/LoopPoint.py

```diff
@@ -0,0 +1,13 @@
+from m5.params import *
+from m5.objects.Probe import ProbeListenerObject
+
+
+class LoopPoint(ProbeListenerObject):
+
+    type = "LoopPoint"
+    cxx_header = "cpu/probes/looppoint.hh"
+    cxx_class = "gem5::LoopPoint"
+
+    target_pc = VectorParam.UInt64(0, "the target PC")
```

Comment:
What does it mean to be 0 here? Is that valid? If not, then we should just have no default.

Reply:
Like the comment above, this should not be the case. Maybe it is, but I think it's a bug in gem5.

I can find plenty of examples both where there is no default for the `Param.<SimObject>` and when there's no default for a `VectorParam`. 

I'm not saying I don't believe you, but it's very strange! :)